components/brave_wallet_ui/utils/account-utils.ts

```diff
@@ -140,3 +141,32 @@ export const findAccountInList = (
     ) ?? account
   )
 }
+
+export const keyringIdForNewAccount = (
+  coin: BraveWallet.CoinType,
+  network?: string | undefined
+) => {
+  if (coin === BraveWallet.CoinType.ETH) {
```

Comment:
Since we have similar code in Android too, could it be moved to the core?

Reply:
I'll try to move to core when bitcoin also gets into this function